mobile_verifier/src/heartbeats/cbrs.rs

```diff
@@ -79,21 +80,27 @@ impl HeartbeatDaemon {
         .await
     }
 
-    async fn process_file(
-        &self,
+    async fn process_file<'a>(
+        &'a self,
         file: FileInfoStream<CbrsHeartbeatIngestReport>,
-        heartbeat_cache: &Cache<(String, DateTime<Utc>), ()>,
-        coverage_claim_time_cache: &CoverageClaimTimeCache,
-        covered_hex_cache: &CoveredHexCache,
+        heartbeat_cache: &Arc<Cache<(String, DateTime<Utc>), ()>>,
```

Comment:
Why the reference? I thought arcs were cheap to copy

Reply:
Fewer lines of code changed :-/